README.md

```diff
@@ -1 +1,200 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
Please remove the `Live Demo` on `line 20.` You only use it when you're building a project that you're going to deploy.